docs/specification.md

```diff
@@ -324,7 +325,7 @@ token_metadata_response =
 
 token_metadata =
```

Comment:
Here and above `token_metadata` type alias is used. I think one of them should be `token_metadata_request`.

Reply:
Also, do we need this entrypoint? Because aren't we implementing this alternative? 

>Contract storage MUST have a big_map that maps token_id -> token_metadata
and annotated %token_metadata

Reply:
Well, If it's required by FA2 so I believe that we should leave it as is here as well

Reply:
But AFAIU FA2 specifies

>Token registry contract MUST implement one of two ways to expose token metadata
for off-chain clients:

So it can be either one of those. I ask because @gromakovsky suggested here [1] that we can use the simpler alternative of just leaving the metadata in the storage, without actually implementing the entrypoint alternative.

[1] https://serokell.slack.com/archives/CUMAQ3PL4/p1592391873179000

Reply:
Alright, have removed this entrypoint completely